var/spack/repos/builtin.mock/packages/adios2/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Adios2(Package):
+    """The Adaptable Input Output System version 2,
+    developed in the Exascale Computing Program"""
```

Comment:
I think it'd be useful to write the description to explain how this package is used in the tests.